VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/EstimatedDecisionDate/EstimatedDecisionDate.test.tsx

```diff
@@ -1,64 +1,42 @@
 import 'react-native'
 import React from 'react'
-// Note: test renderer must be required after react-native.
-import { act, ReactTestInstance } from 'react-test-renderer'
 
+import { screen, fireEvent } from '@testing-library/react-native'
 import { DateTime } from 'luxon'
-
-import { context, mockNavProps, mockStore, render } from 'testUtils'
+import { context, mockNavProps, render } from 'testUtils'
 import { InitialState } from 'store/slices'
 import EstimatedDecisionDate from './EstimatedDecisionDate'
-import { AlertBox, TextView, VAButton } from 'components'
-import { RenderAPI } from '@testing-library/react-native'
 
 const mockExternalLinkSpy = jest.fn()
 
 jest.mock('utils/hooks', () => {
   const original = jest.requireActual('utils/hooks')
-  const theme = jest.requireActual('styles/themes/standardTheme').default
-
   return {
     ...original,
     useExternalLink: () => mockExternalLinkSpy,
   }
 })
 
 context('EstimatedDecisionDate', () => {
-  let store: any
-  let component: RenderAPI
-  let props: any
-  let testInstance: ReactTestInstance
-
   const initializeTestCase = (maxEstDate: string, showCovidMessage: boolean): void => {
-    props = mockNavProps({ maxEstDate, showCovidMessage })
-
-    component = render(<EstimatedDecisionDate {...props} />, { preloadedState: { ...InitialState } })
-    testInstance = component.UNSAFE_root
+    const props = mockNavProps({ maxEstDate, showCovidMessage })
+    render(<EstimatedDecisionDate {...props} />, { preloadedState: { ...InitialState } })
   }
 
-  beforeEach(() => {
-    initializeTestCase('2020-12-20', false)
-  })
-
   it('should initialize', async () => {
```

Comment:
This and the other `async` declarations in the file can be removed

Reply:
removed